src/components/RichTextEditor/RichTextEditor.cy.tsx

```diff
@@ -47,6 +48,28 @@ const LINK_CHOOSER_CHECKBOX = '.tw-group > .tw-inline-flex > .tw-flex-1 > .tw-se
 
 const insertTextAndOpenToolbar = () => cy.get('[contenteditable=true]').click().type('hello{selectall}');
 
+function checkPosition(chainers: string, value: number, text: string) {
```

Comment:
All other functions in this file are declared as const arrow functions. We should keep it consistent and do it the same way here.
`const checkPosition = (chainers: string, value: number, text: string) => { ...`

Reply:
true, thanks!